unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -218,14 +224,26 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
                     forceRender = forceRender,
                 };
 
-                yield return avatar.Load(wearableItems, emotes.ToList(), avatarSettings, loadingCts.Token).ToCoroutine(e =>
-                {
-                    if (e is not OperationCanceledException)
-                        throw e;
-                });
+                yield return avatar.Load(wearableItems, emotes.ToList(), avatarSettings, loadingCts.Token)
+                                   .ToCoroutine(e =>
+                                    {
+                                        if (e is not OperationCanceledException)
+                                            throw e;
+                                    });
             }
 
-            avatar.PlayEmote(model.expressionTriggerId, model.expressionTriggerTimestamp);
+            sceneEmoteHandler.SetExpressionLamportTimestamp(model.expressionTriggerTimestamp);
+
+            if (sceneEmoteHandler.IsSceneEmote(model.expressionTriggerId))
+            {
+                sceneEmoteHandler
+                   .LoadAndPlayEmote(model.bodyShape, model.expressionTriggerId)
```

Comment:
You don't pass a cancellation token

Reply:
Now I see, it's a local instance. Removing `CancellationToken` definitely helped with understanding 👍